notebooks/ts_generic_example.ipynb

```diff
@@ -246,9 +238,94 @@
     "print(run.data)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b0aa46b5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Show the results\n",
+    "\n",
+    "result = run.poll()\n",
+    "print(result.data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "36284c02",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Import Packages for inference\n",
+    "\n",
+    "from tq42.algorithm import (\n",
+    "    GENERIC_ML_INFER,\n",
```

Comment:
for what do we need to have this? 🤔

Reply:
because we did not import them before.
I ll import them all in the beginning